src/main/java/manager/card/Card.java

```diff
@@ -23,8 +23,12 @@ public String getAnswer() {
         return "[A] " + answer;
     }
 
-    public LocalDate getDate() {
-        return date;
+    public LocalDate getDueBy() {
+        return dueBy;
+    }
+
+    public void setDueBy(LocalDate newDueBy) {
+        dueBy = newDueBy;
     }
```

Comment:
@neojiaern can be used in conjunction with my computeNewDeadline functions for revision implementation